lib/internal/main/print_help.js

```diff
@@ -55,10 +55,15 @@ const envVars = new SafeMap(ArrayPrototypeConcat([
     'of stderr' }],
   ['NODE_REPL_HISTORY', { helpText: 'path to the persistent REPL ' +
     'history file' }],
+  ['NODE_REPL_EXTERNAL_MODULE', { helpText: 'path to a Node.js module ' +
+    'which will be loaded in place of the built-in REPL' }],
+  ['NODE_SKIP_PLATFORM_CHECK', { helpText: 'set to 1 to skip ' +
+    'a supported platform during Node.js startup' }],
```

Comment:
```suggestion
    'the check for a supported platform during Node.js startup' }],
```

Reply:
Thank you! done 911f8eb95bb0957def49818807138ca92ea52018